yabt/utils.py

```diff
@@ -52,6 +52,16 @@ def fatal(msg, *args, **kwargs):
     sys.exit(1)
```

Comment:
replace the last 3 lines of this function with a call to `fatal_noexc`

Reply:
Done